### PR TITLE
app-admin/terraform: Bump version to 0.12.3

### PR DESCRIPTION
**Pull request by dmolik**
Package-Manager: Portage-2.3.67, Repoman-2.3.15
Signed-off-by: Dan Molik <dan@danmolik.com>
Tested-by: Dan Molik <dan@danmolik.com>

### CODE REVIEW

File: app-admin/terraform/terraform-0.12.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+src_compile() {
+	cd "src/${EGO_PN}" || die
+	GOPATH="${S}" GOCACHE="${T}/go-cache" go build \
+		-v -work -o "${S}/${PN}" ./ || die
+}
+
+src_install() {
+	dobin terraform
+
+	pushd "src/${EGO_PN}" >/dev/null || die
```
Comment (mgorny) [inline app-admin/terraform/terraform-0.12.3.ebuild]: Nit: why not use the full path in `DOCS`? You can use e.g. `src/${EGO_PN}/{README,CHANGELOG}.md`

File: app-admin/terraform/terraform-0.12.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
[the remark this answers is not in the document]
Comment (dmolik) [reply app-admin/terraform/terraform-0.12.3.ebuild]: Updated!

File: app-admin/terraform/terraform-0.12.3.ebuild (hunk 1 above)
Comment (dmolik) [reply app-admin/terraform/terraform-0.12.3.ebuild]: DOCS changed to:
```
DOCS=( src/${EGO_PN}/{README,CHANGELOG}.md )

**Comment by mgorny** [comment]
Thanks. I'm going to squash the commits, test it and merge if everything is fine.

**Comment by mgorny** [comment]
Oops, squashing ate the `Closes` tag. It's merged as a020555415731c0e46cf2b46311465157c4663d1

**Pull request closed without merging**